packages/payments-client/public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
     <script
       src="https://pi-test.sagepay.com/api/v1/js/sagepay.js"
-      integrity="sha384-DACvrJ5E1mOloI+XEpw6bYlFGYsOI2QhNef+BaLTZkapikXO1DfU2WdPsF20Izj9"
```

Comment:
why has this changed? just wondering if this will be something we have to keep an eye on